src/main/kotlin/no/nav/syfo/personoppgave/PersonOppgaveService.kt

```diff
@@ -44,7 +44,31 @@ class PersonOppgaveService(
         }
     }
 
-    fun behandle(personOppgave: PersonOppgave, veilederIdent: String) {
+    fun behandlePersonOppgaver(
```

Comment:
Hadde kanskje vært bedre om denne kunne gjenbruke koden for å behandle én personoppgave, og at updatePersonoppgaver i databasen bare tok inn oppdaterte personoppgaver og oppdaterte alle i en connection på samme måte som man oppdaterer én oppgave.